PHP/DataManager/CRUDDataOperations.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: CRUD Data Operations
-description: CRUD Data Operations
+title: CRUD Data Operations in PHP DataManager control | Syncfusion
```

Comment:
Control `C` should be caps, check and change in all places.

Reply:
changed